CI/linux/01_install_dependencies.sh

```diff
@@ -32,14 +32,25 @@ install_obs-deps() {
     fi
 }
 
-install_qt-deps() {
+install_qt5-deps() {
     shift
-    status "Install Qt dependencies"
-    trap "caught_error 'install_qt-deps'" ERR
+    status "Install Qt5 dependencies"
+    trap "caught_error 'install_qt5-deps'" ERR
 
     sudo apt-get install -y $@
 }
 
+install_qt6-deps() {
+    shift
+    status "Install Qt6 dependencies"
+    trap "caught_error 'install_qt6-deps'" ERR
+
+    _QT6_AVAILABLE="$(sudo apt-cache search $1)"
```

Comment:
I think `apt-cache madison ${1}` would be the safer variant. At least last time I checked for this use case it was the suggested method.

Reply:
Changed.